blockchain/storagev2/leveldb/leveldb_test.go

```diff
@@ -119,33 +186,28 @@ func TestWriteFullBlock(t *testing.T) {
 	}()
 
 	blockchain := make(chan *types.FullBlock, 1)
-	go storagev2.GenerateBlocks(t, count, blockchain, ctx)
+	go storagev2.GenerateBlocks(t, blockCount, blockchain, ctx)
+
+	readThreads := 3
+	for i := 1; i <= readThreads; i++ {
+		wg.Add(1)
+
+		go readBlock(t, s, blockCount, &wg, ctx)
+	}
 
 insertloop:
-	for i := 1; i <= count; i++ {
+	for i := 1; i <= blockCount; i++ {
 		select {
 		case <-ctx.Done():
 			break insertloop
 		case b := <-blockchain:
-			batchWriter := s.NewWriter()
-
-			batchWriter.PutBody(b.Block.Number(), b.Block.Hash(), b.Block.Body())
-
-			for _, tx := range b.Block.Transactions {
-				batchWriter.PutTxLookup(tx.Hash(), b.Block.Number())
-			}
-
-			batchWriter.PutHeader(b.Block.Header)
-			batchWriter.PutHeadNumber(uint64(i))
-			batchWriter.PutHeadHash(b.Block.Header.Hash)
-			batchWriter.PutReceipts(b.Block.Number(), b.Block.Hash(), b.Receipts)
-			batchWriter.PutCanonicalHash(uint64(i), b.Block.Hash())
-			require.NoError(t, batchWriter.WriteBatch())
-
-			size := dirSize(t, path)
+			writeBlock(t, s, b)
 			t.Logf("writing block %d", i)
-			t.Logf("\tldb file count: %d", countLdbFilesInPath(path))
-			t.Logf("\tdir size %d MBs", size/1_000_000)
 		}
 	}
+
+	size := dirSize(t, path)
+	t.Logf("\tldb file count: %d", countLdbFilesInPath(path))
+	t.Logf("\tdir size %d MBs", size/1_000_000)
```

Comment:
Nitpick, to be more precise (though I'm aware that this code originates prior to this PR), but we need to divide by 1024*1024=1048576
```suggestion
	t.Logf("\tdir size %d MBs", size/1048576)
```

Reply:
Ok, will use leveldb options like this 1*opt.MiB